README.md

```diff
@@ -437,7 +439,31 @@ $ docker run -d -it -p 7200:8000 -p 7201:8001 -p 7202:8002 \
      store/marklogicdb/marklogic-server:10.0-9-centos-1.0.0-ea4
 ```
 
-When you complete these steps, you will have multiple containers; one on each VM and all connected to each other on the 'ml-cluster-network' network. All the containers will be part of same cluster.
+When you complete these steps, you will have multiple containers; one on each VM, and all connected to each other on the 'ml-cluster-network' network. All the containers will be part of same cluster.
+
+# Upgrading to a latest MarkLogic Docker Release
+
+To upgrade MarkLogic Docker from release 10.0-1 to the latest release, perform following steps:
```

Comment:
We need to link back to the existing documentation for upgrades: 
- https://docs.marklogic.com/guide/relnotes/chap2 
- https://docs.marklogic.com/guide/admin/rolling-upgrades 

Something like:
MarkLogic has extensive documentation about upgrades (see https://docs.marklogic.com/guide/relnotes/chap2). Other than the uninstall and install of the MarkLogic RPMs, the overall processes and compatibility notes for upgrades remain the same when you run MarkLogic in containers.  Instead of uninstalling and installing the MarkLogic RPMs, use the following procedure to upgrade a container instance to a newer version of MarkLogic. Be sure to follow sequence described in the documentation for rolling upgrades https://docs.marklogic.com/guide/admin/rolling-upgrades if you need to upgrade with zero downtime.

Reply:
Thanks James. 
Will add the above details and links in the upgrade section.

Reply:
Added suggested info and links to server docs.